src/engine/dom-engine.js

```diff
@@ -906,6 +906,20 @@ $DE.createStatImg = function() {
     return img;
 };
 
+$DE.addFontLinkObject = function(url) {
+    var link = document.getElementById('anm-link');
```

Comment:
Please also use `$doc` here and below instead of `document`. That's not very relevant for a truly long time, but it's the established code of conduct :).